Dockerfile

```
@@ -73,10 +77,12 @@ EOT
 FROM build-base AS build
 ARG BUILD_TAGS
 ARG TARGETPLATFORM
-RUN xx-go --wrap
 RUN --mount=type=bind,target=. \
     --mount=type=cache,target=/root/.cache \
     --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=bind,from=osxcross,src=/osxsdk,target=/xx-sdk \
+    xx-go --wrap && \
+    if [ "$(xx-info os)" == "darwin" ]; then export CGO_ENABLED=1; fi && \
```

Maybe default CGO_ENABLED to 0 on the other platforms? Not sure if it gets set elsewhere.

```suggestion
    export CGO_ENABLED=0 && \
    if [ "$(xx-info os)" == "darwin" ]; then CGO_ENABLED=1; fi && \
```